src/shogun/multiclass/BruteKNNSolver.h

```diff
@@ -16,8 +16,8 @@ namespace shogun
 {
 
 /**
- * The BruteKNNSolver class is a solver class which inherit from KNNSolver class. It will use brute way
- * to classify the object which mean compare the object against all training objects for each prediction.
+ * The BruteKNNSolver class is a solver class and it use brute way to classify the object,
+ * which means test points are compared to all training data for each prediction.
```

Comment:
can you pls use the one I wrote in the comment? 

"Test points are compared to all training data for each prediction."